cluster/gce/gci/configure-helper-common.sh

```diff
@@ -3214,6 +3214,459 @@ function start-cluster-networking {
+        - name: Prefix
+          type: string
+          priority: 0
+          jsonPath: .spec.prefix
+          description: The prefix of the NET CIDR block
+        - name: Vni
+          type: string
+          priority: 0
+          jsonPath: .spec.vni
+          description: The VNI of the VPC
+        - name: Vpc
+          type: string
+          priority: 0
+          jsonPath: .spec.vpc
+          description: The name of the VPC
+        - name: Status
+          type: string
+          priority: 0
+          jsonPath: .spec.status
+          description: The Current Provisioning Status of the net
+        - name: Bouncers
+          type: integer
+          priority: 0
+          jsonPath: .spec.bouncers
+          description: The number of bouncers of the Net
+        - name: CreateTime
+          type: string
+          priority: 0
+          jsonPath: .spec.createtime
+          description: Time the object is created
+        - name: ProvisionDelay
+          type: string
+          priority: 0
+          jsonPath: .spec.provisiondelay
+          description: Time to provision an object from creation
+---
+# mizar CRD vpcs.mizar.com
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: vpcs.mizar.com
+spec:
+  scope: Namespaced
+  group: mizar.com
+  versions:
+    - name: v1
+      served: true
+      storage: true
+  names:
+    kind: Vpc
+    plural: vpcs
+    singular: vpc
+    shortNames:
+      - vpc
+      - vpcs
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          x-kubernetes-preserve-unknown-fields: true
+      additionalPrinterColumns:
+        - name: Ip
+          type: string
+          priority: 0
+          jsonPath: .spec.ip
+          description: The IP of the VPC CIDR block
+        - name: Prefix
+          type: string
+          priority: 0
+          jsonPath: .spec.prefix
+          description: The prefix of the VPC CIDR block
+        - name: Vni
+          type: string
+          priority: 0
+          jsonPath: .spec.vni
+          description: The VNI of the VPC
+        - name: Dividers
+          type: integer
+          priority: 0
+          jsonPath: .spec.dividers
+          description: The number of dividers of the VPC
+        - name: Status
+          type: string
+          priority: 0
+          jsonPath: .spec.status
+          description: The Current Provisioning Status of the net
+        - name: CreateTime
+          type: string
+          priority: 0
+          jsonPath: .spec.createtime
+          description: Time the object is created
+        - name: ProvisionDelay
+          type: string
+          priority: 0
+          jsonPath: .spec.provisiondelay
+          description: Time to provision an object from creation
+EOF
+}
+
 function create-mizar-daemon-manifest {
   cat <<EOF >/etc/kubernetes/manifests/mizar-daemon.yaml
```

Comment:
Same here

Reply:
I tried this .. spent a day getting it to work but I'm missing something. Perhaps someone in arktos team can do this? I need to focus on CNI fixes.

Reply:
Fixed with Sonya's help.